docs/csharp/language-reference/operators/snippets/patterns/patterns.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+    <LangVersion>9.0</LangVersion>
```

Comment:
Do we need to specify `<LangVersion>9.0</LangVersion>`, it defaults to C# 9 when targeting .NET 5 - right?

Reply:
We don't need that. It will default correctly.

Consider adding `<Nullable>enable</Nullable>` for new projects though.